subprojects/gradle/module-dependencies-graph/src/gradleTest/kotlin/com/avito/module/dependencies/AppsDependenciesGraphTest.kt

```diff
@@ -6,7 +6,9 @@ import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.io.TempDir
 import java.io.File
 
-internal class DependenciesGraphTest {
+// TODO: try to get rid of the printing task. It's used for testing only and fragile to changed output
```

Comment:
Do we need this comment?

Reply:
Yes, I want to replace it with integration tests with ProjectBuilder. 
They can check the same scenarios but without checking output.